src/Http/Jobs/JobStatus.php

```diff
@@ -4,20 +4,37 @@
 
 namespace CultuurNet\UDB3\Http\Jobs;
 
-use ValueObjects\Enum\Enum;
-
-/**
- * Class JobStatus
- * @package CultuurNet\UDB3\Http\Jobs
- * @method static JobStatus WAITING()
- * @method static JobStatus RUNNING()
- * @method static JobStatus FAILED()
- * @method static JobStatus COMPLETE()
- */
+use CultuurNet\UDB3\Model\ValueObject\String\Enum;
+
 class JobStatus extends Enum
 {
-    public const WAITING = 'waiting';
-    public const RUNNING = 'running';
-    public const FAILED = 'failed';
-    public const COMPLETE = 'complete';
+    public static function getAllowedValues(): array
+    {
+        return [
+            'waiting',
+            'running',
+            'failed',
+            'complete',
+        ];
+    }
+
+    public static function WAITING(): JobStatus
```

Comment:
I would prefer to use the lower case function names here. Then we follow the naming convention of functions.